Assets/MultiBuild/Editor/SettingsWindow.cs

```diff
@@ -266,10 +283,10 @@ public class SettingsWindow : EditorWindow {
 
         void UpdateTargetsNotAdded() {
             _targetNamesNotAdded = new List<string>();
-            foreach (var target in Targets) {
-                if (!Settings.targets.Contains(target)) {
+            for (int i = 0, n = (int)Target.Max; i < n; ++i) {
+                Target target = Targets[i];
+                if (!Settings.targets.Contains(target))
```

Comment:
Sorry just realised this change is wrong too. Maybe you're trying to avoid the gc overhead of the enumerator (personally since this is in the editor, and even then not in a loop I don't think it matters anyway), but you're using `n = Target.Max` as the high value and using `i < n`, which means the last item is never processed (Max is the last index, not Count).

Personally I think the `foreach` was fine in this case.